NEL.MESH/Services/Mesh/MeshService.cs

```diff
@@ -61,7 +62,36 @@ public MeshService(IMeshBroker meshBroker)
             });
 
         public ValueTask<Message> SendFileAsync(Message message) =>
-                throw new System.NotImplementedException();
+            TryCatch(async () =>
+            {
+
+                ValidateMeshMessageOnSendFile(message);
+                HttpResponseMessage responseFileMessage = await this.meshBroker.SendFileAsync(
+                        mailboxTo: message.Headers["Mex-To"].First(),
```

Comment:
```suggestion
localId: GetKeyStringValue("Mex-To", message.Headers)
```
also make this change on the others